openslides/motions/migrations/0003_motion_comments.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from ...core.config import config
+
+
+def change_motion_comments(apps, schema_editor):
+    """
+    Index the comments fields in the config. Changing from an array to a dict with the
+    ids as keys. CHange all motions from an array for comments to a dict with the comments
+    field id as key to link motion comments and comments fields.
+    """
+    # We get the model from the versioned app registry;
+    # if we directly import it, it will be the wrong version.
+    Motion = apps.get_model('motions', 'Motion')
+
+    config_comments_fields = config['motions_comments']
```

Comment:
@normanjaeckel I wonder why this works here to use the ConfigHandler?  Is this migration OK for you or should I do it different?

Reply:
I am not sure but maybe we should use `apps.get_model('core', 'ConfigStore')` to be more save.